docs/Coding-Guidelines.md

```diff
@@ -57,3 +57,66 @@ ABI.
 GitHub diff is line oriented. Keeping lines within the preferred limit makes changes easier to
 review. Use a tool like Prettier to bulk-reformat files, or configure your editor's "rulers." If new
 languages become common, we will describe the coding guidelines for such languages here.
+
+## Git Checkin-to-main Policy
+
+The main branch needs to be kept 'healthy' and ready to ship near daily (daily, or within a day or
+two). An unhealthy main branch negatively impacts day-to-day development and jeopardizes release
+schedules. We have policies to ensure a high quality bar in main. The details differ depending if a
+code change is Experimental vs Preview or Stable.
+
+See [Final Decision: Release channels and cadences #637](https://github.com/microsoft/WindowsAppSDK/discussions/637)
+for the expectations of the various release channels.
+
+### Policy: Preview and Stable (aka non-Experimental)
+
+The following requirements must be met to checkin non-Experimental changes to main:
+
+1. API Review approval
+2. Tests
```

Comment:
What kind of tests? For example, automated vs. manual? Do they need to be part of the build? Do they need to run?

Reply:
Whatever level of tests be appropriate.

We have unit+functional + integration tests, and automated + manual. Specific mix is up to each feature team's judgement.